src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private Operation operation;

```

Comment:
Since the `operation` field can be changed through the setter method `setOperation`, it is not consistent to have `fruit` and `quantity` as final. If the intention is to make `FruitTransaction` immutable, you should remove the setter method and declare all fields as final, otherwise, you should remove the `final` keyword from `fruit` and `quantity` to maintain consistency.